docs/docs-freshness-registry.json

```
@@ -2,6 +2,49 @@
+      "last_review": "2026-04-14",
+      "cadence_days": 30
+    },
+    {
+      "path": "tasks/specs/linear-a560f416-c80a-473b-96fb-9f1ec090c968.md",
+      "owner": "Codex (top-level agent), Review agent",
+      "status": "active",
+      "last_review": "2026-04-14",
+      "cadence_days": 30
+    },
+    {
+      "path": "tasks/tasks-linear-a560f416-c80a-473b-96fb-9f1ec090c968.md",
```

Biome (2.4.10)

[error] The key path was already declared.

(lint/suspicious/noDuplicateObjectKeys)